kflynn/emissary
# Land 2.0.4 content into release/v2.0

Land 2.0.4 content into release/v2.0.

- dtest_k3s: Log to dlog
- dtest_k3s: Better error handling and logging
- dtest_k3s: Update to work with cgroups v2
- Delete pkg/helm, clean up go.mod
- rm -rf internal pkg/teleproxy teleproxy-old && git checkout HEAD internal/pkg/testutils && go mod tidy && go mod vendor && make $PWD/OPENSOURCE.md
- rm -rf build-aux/.circleci/ build-aux/CHANGELOG.md build-aux/tests/teleproxy.bats
- Delete build-aux/go-mod.mk and associated
- rm _cxx/go.mod
- rm -rf cmd/k3sctl cmd/kubeapply
- builder.mk: Begone with e2etest
- Update MAINTAINERS.md
- Shut up failures from no-longer-used CI systems
- Makefiles: Fix dependencies around ambassador.version
- build-system: Remove the no-longer-used -rc-latest stuff
- Consolidate build tools in to `./tools/src/`, Telepresence-style
- Revert "just remove the dumb CI check"
- Edit ambassador.version when promoting an image
- Switch to use github.com/datawire/dtest
- fix broken chart publish script
- more CI updates
- fix push_chart.sh
- Update for v2.0.4
- docs/CHANGELOG.tpl: Trim trailing whitespace
- CI: Check that the Envoy version has been pushed to the right places
- Fix .dockerignore
- builder: Be clearer about `docker build` commands
- DEVELOPING.md: Add a section about debugging should-not-happen-in-CI fails
